Assets/MixedRealityToolkit.Extensions/Sharing/SpatialAlignment/QR.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
+guid: 1f2ff87d1bbb54148a536756bcfd486b
```

Comment:
if nothing is in these directories, they probably aren't worth adding yet.

If its felt that these directories need to exist, i would prefer if this were called QRCode compared to QR